src/components/local/exchange/InstrumentSelectA.js

```diff
@@ -26,21 +27,95 @@ export default class InstrumentSelectA extends React.Component {
     selectImage(type) { //get Image by sortType
         const sortType = this.state.sortType;
         if (sortType.indexOf(type) < 0) {
-            return 'updown_1.png';
+            return updown_1;
         } else if (sortType.indexOf('up') > -1) {
-            return 'updown_2.png';
+            return updown_2;
         }
-        return 'updown_3.png';
+        return updown_3;
     }
 
     selectSort(type) { //sortType change
         if (this.state.sortType === `up-${type}`) {
-            this.setState({ sortType: `down-${type}` });
+            this.setState({sortType: `down-${type}`});
         } else {
-            this.setState({ sortType: `up-${type}` });
+            this.setState({sortType: `up-${type}`});
         }
     }
 
+    doSort(type, coin) {
+        if (!coin) {
+            return [];
+        }
+        // const volume = this.state.volume;
```

Comment:
If we do not need a reference to the old code then please remove the commented code

Reply:
I deleted it. Thank you